openvas/Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN set -ex; \
     apt update; \
     DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends ospd-openvas openvas python3-pkg-resources rsync wget openvas-smb nmap; \
     sed -i 's|/var/log/gvm/openvas.log|/dev/stdout|g' /etc/openvas/openvas_log.conf; \
-    sed -i 's/if \[ "`id -u`" -eq "0" \]/if fasle \&\& [ "`id -u`" -eq "0" ]/' /usr/bin/greenbone-nvt-sync; \
+    sed -i 's/if \[ "`id -u`" -eq "0" \]/if fasle \&\& [ "`id -u`" -eq "0" ]/' /usr/bin/greenbone-nvt-sync --rsync; \
```

Comment:
This will pass `--rsync` to `sed` command and throw an error message. The `/usr/bin/greenbone-nvt-sync` in this line is just the path of the file that we are modifying with `sed`